Integrations/ZeroFox/ZeroFox.py

```diff
@@ -0,0 +1,579 @@
+    context: Dict = {'ZeroFox.Alert(val.ID && val.ID === obj.ID)': contents}
+    return_outputs(
+        tableToMarkdown(f'ZeroFox Alert {alert_id}', contents_war_room, removeNull=True),
+        context,
+        response_content
+    )
+
+
+def create_entity(name: str, strict_name_matching: bool = None, image: str = None, labels: list = None,
+                  policy: int = None, organization: str = None):
+    url_suffix: str = '/entities/'
+    request_body: Dict = {
+        'name': name,
+        'strict_name_matching': strict_name_matching,
+        'image': image,
+        'labels': labels,
+        'policy': policy,
+        'organization': organization
+    }
+    request_body: Dict = remove_none_dict(request_body)
+    response_content: Dict = http_request('POST', url_suffix, data=json.dumps(request_body))
+    return response_content
+
+
+def create_entity_command():
+    name: str = demisto.args().get('name')
+    strict_name_matching: bool = bool(demisto.args().get('strict_name_matching'))
+    image: str = demisto.args().get('image')
+    labels: str = demisto.args().get('args')
+    labels: List = argToList(labels, ',')
+    policy: int = int(demisto.args().get('policy'))
+    organization: str = demisto.args().get('organization')
+    response_content: Dict = create_entity(name, strict_name_matching, image, labels, policy, organization)
+    if not response_content or not isinstance(response_content, Dict):
+        raise Exception('Unexpected outputs from API call.')
+    entity_id: int = response_content.get('id')
+    return_outputs(
+        f'Entity has been created successfully. ID: {entity_id}',
+        {'ZeroFox.Entity(val.ID && val.ID === obj.ID)': {'ID': entity_id}},
+        response_content
+    )
+
+
+def list_alerts(params: Dict):
+    url_suffix: str = '/alerts/'
+    response_content: Dict = http_request('GET', url_suffix, params=params)
+    return response_content
+
+
+def list_alerts_command():
+    params: Dict = remove_none_dict(demisto.args())
+    response_content: Dict = list_alerts(params)
+    if not response_content:
+        return_outputs('No alerts found.', outputs={})
+    elif isinstance(response_content, Dict):
+        alerts: List = response_content.get('alerts')
+        contents: Dict = [get_alert_contents(alert) for alert in alerts]
+        contents_war_room: Dict = [get_alert_contents_war_room(content) for content in contents]
+        context: Dict = {'ZeroFox.Alert(val.ID && val.ID === obj.ID)': contents}
+        return_outputs(
+            tableToMarkdown('ZeroFox Alerts', contents_war_room, removeNull=True),
+            context,
+            response_content
+        )
+    else:
+        return_outputs('Unexpected outputs from API call.', outputs={})
+
+
+def list_entities(params: Dict):
+    url_suffix: str = '/entities/'
+    response_content: Dict = http_request('GET', url_suffix, params=params)
+    return response_content
+
+
+def list_entities_command():
```

Comment:
should be able to filter only by whats available in the ui and all filters should be by human readable and not IDs